lib/livebook_web/live/hub/edit/personal_component.ex

```diff
@@ -33,6 +33,23 @@ defmodule LivebookWeb.Hub.Edit.PersonalComponent do
   def render(assigns) do
     ~H"""
     <div id={"#{@id}-component"} class="space-y-8 pb-8">
+      <.modal
```

Comment:
Just curious: is there any DOM reason you moved modal to the top?

Reply:
The overlay had a top margin because of the `space-y-*` tailwind class

Reply:
Ah, we should probably have it outside. We need a single root element though, so perhaps:

```
<div>
  <div ...> content </div>

  <.modal ...> ...</.modal>
</div>
```